finetuning/.gitignore

```diff
@@ -4,3 +4,4 @@ sam_embeddings/
 results/
 iterative_prompting_results/
 *.sh
+run_all_finetuning.py
```

Comment:
Why do we add this to the gitignore? If it's a script shouldn't we just add it?

Reply:
I wasn't sure if I should add it. It's a script to run all the finetunings at once. I added it just now!